proto/qdrant/cloud/cluster/backup/v1/backup.proto

```diff
@@ -635,6 +635,8 @@ message BackupRestore {
   BackupRestoreStatus status = 6 [(buf.validate.field).enum.defined_only = true];
   // The timestamp when the backup restore was deleted (or when deletion started).
   google.protobuf.Timestamp deleted_at = 7;
+  // If set, backup should be restored into a new cluster.
+  optional bool create_new_cluster = 8;
```

Comment:
what will be the cluster-id, this is a required UUID?

Reply:
The cluster-id will be either existing or new cluster-id, this bool makes sure that we don't error out if cluster-id does not exist but we need to create a new cluster. This was made explicit so that a typo in cluster-id would not end up in a new cluster without explicitly setting this bool.

- If cluster-id does not exist and create_new_cluster is False -> error out, cluster-id does not exist
- If cluster-id does not exist and create_new_cluster is True -> create new cluster